benchmark/operations/create_container.py

```diff
@@ -39,11 +43,14 @@ def find_match(states):
 
         return d
 
-    return loop_until(reactor, state_reached)
+    d = loop_until(reactor, state_reached)
+    _timeout(reactor, d, timeout)
```

Comment:
I wanted this to be based on time, since the main cause of delay is the Docker image pull, which I guess should usually take < 10 minutes.  And if experience says not, I'd like to be able to ensure we allow more time, not just more retries.